docs/se/config/advanced-configuration.adoc

```diff
@@ -259,37 +255,28 @@ sources gives your application a way to access all config elements distinctly ev
 if their keys would otherwise conflict.
 
 ===== Merging Strategies
-The `ConfigSources.create(Supplier<ConfigSource>...)` and `ConfigSources.create(List<Supplier<ConfigSource>...)`
-methods return a `CompositeBuilder`.
-By default, earlier sources in the list have higher priority than later ones, meaning
+When creating config from multiple sources it is possible, that the same key comes from multiple
+sources. By default, earlier sources in the list have higher priority than later ones, meaning
 that if the same key appears in two or more sources the source earlier in the
```

Comment:
By default, earlier sources in the list have higher priority than later ones. This means that if the same key appears in two or more sources, then the source earlier in the list prevails.